test/mock_schema_delegate.js

```diff
@@ -20,6 +20,7 @@ function makeSchemaFunctionDef(functionType, functionName, schema, isMeta) {
         const argname = schema.args[i];
         const argrequired = !!schema.required[i];
         const arginput = !!schema.is_input[i];
+        const argunique = schema.is_unique ? !!schema.is_unique[i] : false;
```

Comment:
Yeah time to ditch this messy JSON format. Just import whatever i did on almondscript...